src/workerd/api/streams/standard.c++

```diff
@@ -2067,6 +2067,14 @@ void ReadableStreamJsSource::cancel(kj::Exception reason) {
 }
 
 void ReadableStreamJsSource::doClose() {
+  if (readPending) {
+    // If we're closed in the middle of a read, we do not want to actually
+    // change the state yet we still have data to receive and process. Instead,
```

Comment:
Should "... yet we still have data to receive ..." be "... yet, because we still have data to receive ..."?